contracts/treasury/ERC20Treasury.sol

```diff
@@ -5,12 +5,14 @@ import "@openzeppelin/contracts/utils/Context.sol";
 import "../interfaces/ERC20DepositStrategy.sol";
 import "../interfaces/ERC20WithdrawStrategy.sol";
 
+import "../Version.sol";
+
 /**
  * @title Vault for ERC20 token types.
  *
  * @dev
  */
-contract ERC20Treasury is Context {
+contract ERC20Treasury is Version, Context {
```

Comment:
Need to inherit Version into all other applicable contracts.

Reply:
Excellent, that is exactly what I'd expect 😁

Reply:
doing all the inheritance in another contract